applications/crossbar/src/modules/cb_call_inspector.erl

```diff
@@ -197,25 +196,23 @@ get_cdr_ids(Db, View, ViewOptions) ->
 
 -spec filter_cdr_ids(kz_proplist()) -> kz_proplist().
 filter_cdr_ids(Ids) ->
-    Req = [{<<"Call-IDs">>, [CallId || {<<_:7/binary, CallId/binary>>, _} <- Ids]}
+    %% Remove leading year, month and dash
+    CallIds = [CallId || {<<_:7/binary, CallId/binary>>, _} <- Ids],
```

Comment:
Maybe make a macro to pattern-match the YYYY, MM, and CallId out to make this more clear than a comment?